config/config.go

```diff
@@ -731,6 +731,8 @@ func loadConfigFromFile(configFile string) (c Local, err error) {
 	}
 
 	// Migrate in case defaults were changed
+	// If a config file does not have version, it is assumed to be zero.
+	// All fields listed in migrate() might be changed if an actual value matches to default value from a previous version.
```

Comment:
1. No logger exists to produce a message
2. Can't assume the latest version because missed version means 0.